src/GAP.jl

```diff
@@ -59,12 +59,7 @@ include("setup.jl")
 import Libdl
 import Random
 
-# setup the initial sysinfo dictionary; we'll update this later in __init__
-# this also ensures that Setup.regenerate_gaproot gets precompiled, reducing
-# the startup time a little bit
-const sysinfo = Setup.regenerate_gaproot()
-
-const GAP_VERSION = VersionNumber(sysinfo["GAP_VERSION"])
+const GAP_VERSION = VersionNumber(Setup.read_sysinfo_gap(joinpath(GAP_jll.find_artifact_dir(), "lib", "gap", "sysinfo.gap"))["GAP_VERSION"])
```

Comment:
I actually have written code in the past that used `GAP.sysinfo`. From that POV it would be nice to at least retain it, e.g.
```suggestion
const sysinfo = Setup.read_sysinfo_gap(joinpath(GAP_jll.find_artifact_dir(), "lib", "gap", "sysinfo.gap"))

const GAP_VERSION = VersionNumber(sysinfo["GAP_VERSION"])
```

Reply:
The difference would be that `GAP.sysinfo` used to be the sysinfo with updated fields that was created during startup, but with this change it would instead be the sysinfo that is bundled with `GAP_jll`.